webapp/src/App.vue

```diff
@@ -183,15 +183,19 @@ export default {
 			const isExclusive = module => module.type === 'call.bigbluebutton' || module.type === 'call.zoom'
 			if (!this.$mq.above.m) return // no background rooms for mobile
 			if (this.call) return // When a DM call is running, we never want background media
+			const newRoomHasMedia = newRoom && newRoom.modules && newRoom.modules.some(module => mediaModules.includes(module.type))
 			if (oldRoom &&
 				this.rooms.includes(oldRoom) &&
 				!this.backgroundRoom &&
 				oldRoom.modules.some(module => mediaModules.includes(module.type)) &&
 				this.$refs.primaryMediaSource.isPlaying() &&
 				// don't background bbb room when switching to new bbb room
-				!(newRoom?.modules.some(isExclusive) && oldRoom?.modules.some(isExclusive))
+				!(newRoom?.modules.some(isExclusive) && oldRoom?.modules.some(isExclusive)) &&
```

Comment:
Look OK, though the `?` in `oldRoom?.modules.some(isExclusive)` is redundant, because the `oldRoom` has been guarded to be non-falsy at line 187 (`if (oldRoom`).